packages/cf-component-tabs/src/Tabs.js

```diff
@@ -64,7 +64,9 @@ class Tabs extends React.Component {
             })}
           </ul>
         </Viewport>
-        {this.props.children}
+        {this.props.children.find(child => {
```

Comment:
Be careful when dealing with `this.props.children`, it's perfectly valid for it to be a string. See https://facebook.github.io/react/docs/react-api.html#react.children for more info.